### ISSUE TITLE
Finalizing "signal strength"-feature

### ISSUE THREAD

**Issue by codingwithelijah** [amosproj/amos2022ss05-find-my-hearing-aid#94]
## User story
1. As a user
2. I want / need to be able to navigate to my BLE device by using the "signal strength"-feature with a understandable GUI
3. So that I get real value from using the app

## Further description
precondition: 
* #49 
* #71 
* #74 
* #75

## Acceptance criteria
* Make sure that all parts from precondition-issues are connected correctly
* User should be able to first see devices in his area, then select one, and then be directed to the "signal strength"-feature (correct handling of different views important)
* User is now able to use the "signal strength"-feature without additional help

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
